android/tangram/jni/platform_android.cpp

```diff
@@ -302,7 +302,10 @@ void featureSelectionCallback(JNIEnv* jniEnv, const std::vector<Tangram::TouchIt
     jresult = reinterpret_cast<jlong>(new std::shared_ptr<Tangram::Properties >(items[0].properties));
     jobject object = jniEnv->NewObject(propertiesClass, propertiesConstructorMID, jresult, true);
 
-    jniEnv->CallVoidMethod(tangramInstance, featureSelectionCbMID, object);
+    jdoubleArray itemCenter = jniEnv->NewDoubleArray(2);
```

Comment:
Nopes, I think if you use the `*Region` call then you do not need to do a Release.

http://developer.android.com/training/articles/perf-jni.html#region_calls